minvws/nl-kat-coordination
# Fix add related, fix manual ooi task list, remove redundant octopoes call

### Changes

Two fixes:
- Add related now actually adds a related object instead of same ooi type
- Manually added scans show up correctly in list
- Fixes some view inheritance, increasing ooi detail speed by removing unneeded octopoes request

### Demo

<img width="738" alt="image" src="https://github.com/user-attachments/assets/ef4ade7e-b0da-4e7a-a8a7-f0685858a133">

<img width="1064" alt="image" src="https://github.com/user-attachments/assets/cc40e152-8fe3-4025-95d4-9515966050d0">



### QA notes

Add related: go to ooi detail and click "Add" under the related objects. Show now work again.

After adding a raw file, in the ooi detail, the observation should now show: Scan was added manually.

Whole ooi detail should be QA'ed because the view inheritance has changed so there is a chance of regressions.


---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
